tests/phpunit/tests/test-install.php

```diff
@@ -47,6 +47,7 @@ public function test_uninstall_removing_data() {
 		$english = self::$model->get_language( 'en' );
 
 		self::create_language( 'fr_FR' );
+		$french = self::$model->get_language( 'fr' );
```

Comment:
Why is it necessary? 
If it's to refresh the languages list, it should has already been done at the end of `add_language()` right?

Reply:
Only to use it here: https://github.com/polylang/polylang/blob/87e631ce277ef4aa93d556a07f2e8e110750f2fa/tests/phpunit/tests/test-install.php#L131

Reply:
Oops 😅